LandSandBoat/server
# [Quest] Convert Distant Loyalties to IF

**_I affirm:_**

## What does this pull request do?

Converts the quest Distant Loyalties to IF

## Steps to test these changes

Zone: Southern San 'dOria !zone 230
-- Femitte/Rouva !pos -17.7129 2.1000 10.1636
Talk to Rouva to test the default action
Talk to Femitte to test the default action

Set your fame level to 4 or higher for San d'Oria
!setfamelevel 0 4 <player>

Talk to Femitte and select the first option to start the quest.

Talk to Femitte and you should receive the default action text.

Talk to Rouva and you should receive the quest default text.

Zone to Bastok Markets
-- Bastok Markets !zone 235
Talk to Michea: !pos -299.2114 -15.9717 -156.0629
Trade Michea a Mythril Ingot !giveitem 653
Talk to Michea to test the quest default text.
Rezone !goto <me>
Talk to Michea to receive KI

Zone to Southern San d'Oria !zone 230
-- Femitte/Rouva !pos -17.7129 2.1000 10.1636
Talk to Rouva, her text should not have changed and should still tell you to go to Bastok
Talk to Femitte to end the quest
Talk to Rouva to test that she has reverted to her default action

Zone to Bastok Markets
-- Bastok Markets !zone 235
Talk to Michea: !pos -299.2114 -15.9717 -156.0629
Recieve one time flavor text for the quest. Afterward, she will revert to her default action from then on.

##Captures
Distant Loyalties Quest:
https://www.youtube.com/watch?v=Q4pDVbXe7iU
https://drive.google.com/drive/folders/1VRACiqRlw0vZEO6hp3DXr95rdJQlCi0N?usp=sharing

Default Actions:
https://www.youtube.com/watch?v=8WAJx6_sjGk
https://drive.google.com/drive/folders/1WY6GYNAZMP5J0iw1Pfdxzf4QQXADn8pK?usp=sharing
